includes/tasksProcess/formScripts.js

```diff
@@ -0,0 +1,104 @@
+function getStartDateTask(value) {
+  var x = (document.getElementById("due_date").min = value);
+}
+
```

Comment:
For example- add a general description :" // form validation "

Reply:
done.